regolith/helpers/a_projectumhelper.py

```diff
@@ -154,6 +160,10 @@ def db_updater(self):
             pdoc.update({
                 'collaborators': [],
             })
+        if rc.notes:
```

Comment:
if you set a default, is this if needed?

Reply:
I removed the 'if'. Also, Textarea seems good when I checked it in helper_gui